anndata/tests/helpers.py

```diff
@@ -335,6 +337,13 @@ def assert_equal_h5py_dataset(a, b, exact=False, elem_name=None):
     assert_equal(b, a, exact, elem_name=elem_name)
 
 
+@assert_equal.register(da.Array)
+def assert_equal_dask_array(a, b, exact=False, elem_name=None):
+    from dask.array.utils import assert_eq
+
+    assert_eq(a, b, check_dtype=False)
```

Comment:
```suggestion
assert da.equal(a,b).all()
```
I would just go with this so that the azure tests pass

Reply:
Is this a python version thing? No need to worry about that, since we'll drop 3.7 in the release this is going into.

Reply:
Implementing this in #820

Reply:
Ok then we are good I think